test/metabase/api/card_test.clj

```diff
@@ -367,10 +367,15 @@
 (deftest save-card-with-empty-result-metadata-test
   (testing "we should be able to save a Card if the `result_metadata` is *empty* (but not nil) (#9286)"
     (mt/with-model-cleanup [Card]
-      (= :wow
-         (mt/user-http-request :rasta :post 202 "card" (assoc (card-with-name-and-query)
-                                                              :result_metadata    []
-                                                              :metadata_checksum  (#'results-metadata/metadata-checksum [])))))))
+      (is (= :wow
+             (mt/user-http-request
+              :rasta
+              :post
+              202
+              "card"
+              (assoc (card-with-name-and-query)
+                     :result_metadata    []
+                     :metadata_checksum  (#'results-metadata/metadata-checksum []))))))))
```

Comment:
one arg per line looks a little goofy here, but :shrug:

Reply:
I think it would have overrun the line length limit.  At any rate, this entire test has just been rewritten so we'll see if the new formatting seems OK.